src/components/pages/TeachersPage/Chatbox.tsx

```diff
@@ -6,11 +6,14 @@ import chatboxCSS from './Chatbox.css';
 import Conversation from './Conversation';
 import CopyButton from '@components/shared/CopyButton';
 
-export default function Chatbox({ chat }) {
+export default function Chatbox({ chat, showCopyButton, selected }) {
```

Comment:
I think we should rename `showCopyButton` to `isTheDisplayedChat`.  Its more self-explanatory. Right now, show copy button is the only difference. But in the future, there might be more differences between the main displayed chat and the other chats on the teachers page. This new variable name will remain relevant for those future differences.

Reply:
@mssiegel  do you mean rename `selected` to `isTheDisplayChat` ?

Reply:
Oh, good point. I'd forgotten that `selected` prop was there!  In that case, yes I think we should rename 'selected' to either `isTheDisplayedChat` or `isTheSelectedChat` because its more self-explanatory.    And I don't think we need `showCopyButton` because we could determine to show the copy button if its the selected chat.

Reply:
Actually, I just realized I I will need two seperate variables to be able to apply the `scrollDown` only to the top displayed chat but not to the bottom one. 

I think if I rename them `isTheDisplayedChat` and `inAllStudentChatDisplay` I would be able to make this work and are more descriptive names.